components/brave_ads/core/internal/ml/data/vector_data.cc

```diff
@@ -5,13 +5,15 @@
 
 #include "brave/components/brave_ads/core/internal/ml/data/vector_data.h"
 
+#include <cstddef>
 #include <limits>
 #include <numeric>
 #include <utility>
+#include <vector>
 
+#include "base/check.h"
 #include "base/check_op.h"
```

Comment:
We already include check.h, if we do not use operator checks then we can remove check_op.h

Reply:
we use both `DCHECK` and `DCHECK_LT`

Reply:
only `check_op.h` right?

Reply:
See my other comment on what to include, thanks

Reply:
if I use both it seems like `check_op` is the one to include, per clangd warnings. Wanted to double check with you

Reply:
The rule is if you use `DCHECK` then always only include `check.h`, but if you only use `DCHECK_##` then only include `check_op.h`

Reply:
I'm confused. Let me recap: We use both `DCHECK` and `DCHECK_##`, by what you said in the previous message I should _only_ include `check.h`, but that doesn't seem right because `check_op.h` includes `check.h` and not viceversa (also clang complains if I only include `check.h` because the `DCHECK_##` is not covered).